cmd/syncthing/gui.go

```diff
@@ -188,16 +210,15 @@ func startGUI(cfg config.GUIConfiguration, assetDir string, m *model.Model) erro
 		ReadTimeout: 10 * time.Second,
 	}
 
-	csrv := &folderSummarySvc{model: m}
-	go csrv.Serve()
+	s.fss.ServeBackground()
 
-	go func() {
-		err := srv.Serve(listener)
-		if err != nil {
-			panic(err)
-		}
-	}()
-	return nil
+	err := srv.Serve(s.listener)
+	l.Warnln("API:", err)
```

Comment:
Should we check the value of err here? Given the service might be stopped gracefully?